tests/network/general/test_network_naming.py

```diff
@@ -12,6 +12,7 @@ def invalid_network_names():
 
 
 @pytest.mark.polarion("CNV-8304")
+@pytest.mark.gating
 def test_vm_with_illegal_network_name(namespace, unprivileged_client, invalid_network_names):
```

Comment:
Why do we want this scenario as a gating?

Reply:
We probably need to set a proper criteria for gating. I tend to agree that this test is not covering an important represebtative feature. Lets follow up on this.

Reply:
If there are already 2 team members that are reluctant about  having this test as a gating, then I am good with removing it.